Error: The `form` element was not requested.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`. Note: If radio inputs are wrapped in their `label`, `br` is unnecessary.
diff --git a/output/Radio Group/1/c7861511-2c63-4dbd-a603-0a1a7ba956f9.html b/output/Radio Group/1/c7861511-2c63-4dbd-a603-0a1a7ba956f9.html
index 64432ba..aac94c5 100644
--- a/output/Radio Group/1/c7861511-2c63-4dbd-a603-0a1a7ba956f9.html	
+++ b/output/Radio Group/1/c7861511-2c63-4dbd-a603-0a1a7ba956f9.html	
@@ -1,16 +1,15 @@
-<form>
-  <fieldset>
-    <legend>Choose a theme:</legend>
-    <label>
-      <input type="radio" name="theme" value="light"> Light
-    </label>
-    <br>
-    <label>
-      <input type="radio" name="theme" value="dark"> Dark
-    </label>
-    <br>
-    <label>
-      <input type="radio" name="theme" value="high-contrast"> High Contrast
-    </label>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>Choose a theme:</legend>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light">
+    Light
+  </label>
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark">
+    Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast">
+    High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
